shared/settings/logout/index.tsx

```diff
@@ -28,10 +29,16 @@ const HoverBox = Styles.isMobile
 
 class LogOut extends React.Component<Props, State> {
   state = {
+    loggingOut: false,
     password: '',
     showTyping: false,
   }
 
+  logOut = () => {
+    this.props.onLogout()
+    this.setState({loggingOut: true})
```

Comment:
I'd love to take GUI review but I cannot judge whether we should do this instead of having this in redux state and setting it in reducers. cc @keybase/react-hackers please review